### PR TITLE
use server scaling for xsession

### PR DESCRIPTION
**Pull request by ksimuk**
https://codio.myjetbrains.com/youtrack/issue/codio-14123/Can-we-improve-the-way-Turtle-Graphics-and-other-things-that-run-in-VNC-work-in-Codio

* switch to tiger vnc for server side scaling support
* set default for server side scaling for lite.vnc

### CODE REVIEW

**Review by destitutus — DISMISSED** [review]

File: xserver/files/vnc.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Service]
+User = codio
+Group = codio
+Type=forking
+ExecStart = /usr/bin/vncserver :0  -verbose -SecurityTypes None -auth /tmp/codio.auth -xstartup /usr/bin/openbox
+ExecStop = /usr/bin/vncserver -kill
```
Comment (destitutus) [inline xserver/files/vnc.service:6]: it will not work, display port option is required, `/usr/bin/vncserver -kill :0`

Comment (destitutus) [reply xserver/files/vnc.service]: Maybe it is depends on vncserver, but for tightvnc I see the behaviour

```
➜  new-generation /usr/bin/vncserver -kill
TightVNC Server version 1.3.10

Usage: vncserver [<OPTIONS>] [:<DISPLAY#>]
       vncserver -kill :<DISPLAY#>

<OPTIONS> are Xtightvnc options, or:

        -name <DESKTOP-NAME>
        -depth <DEPTH>
        -geometry <WIDTH>x<HEIGHT>
        -httpport number
        -basehttpport number
        -alwaysshared
        -nevershared
        -pixelformat rgb<NNN>
        -pixelformat bgr<NNN>

See vncserver and Xtightvnc manual pages for more information.
➜  new-generation /usr/bin/vncserver -kill :*
zsh: no matches found: :*
➜  new-generation /usr/bin/vncserver -kill ":*"

Can't tell if :* is on fs-new
Use -kill :<number> instead

➜  new-generation /usr/bin/vncserver :0

You will require a password to access your desktops.

Password:
Password too short
➜  new-generation /usr/bin/vncserver :0

You will require a password to access your desktops.

Password:
Verify:
Would you like to enter a view-only password (y/n)? n
xauth:  file /home/vagrant/.Xauthority does not exist

New 'X' desktop is fs-new:0

Creating default startup script /home/vagrant/.vnc/xstartup
Starting applications specified in /home/vagrant/.vnc/xstartup
Log file is /home/vagrant/.vnc/fs-new:0.log

➜  new-generation /usr/bin/vncserver -kill ":*"

Can't tell if :* is on fs-new
Use -kill :<number> instead

➜  new-generation /usr/bin/vncserver -kill :1

Can't find file /home/vagrant/.vnc/fs-new:1.pid
You'll have to kill the Xtightvnc process manually

➜  new-generation /usr/bin/vncserver -kill :0
Killing Xtightvnc process ID 1242315
```

`/usr/bin/vncserver -kill :*` does not work for me

Comment (destitutus) [reply xserver/files/vnc.service]: did you check if command works with installed software?

Comment (ksimuk) [reply xserver/files/vnc.service]: the playbook is using tigervnc not tightvnc
 systems is not executing the command with a shell, so no wildcards processing
yes I tested the command

Comment (destitutus) [reply xserver/files/vnc.service]: ok, sounds good! 👍

**Pull request merged**